src/modules/pua_json/defs.h

```diff
@@ -139,4 +147,12 @@
 </dialog>\
 </dialog-info>"
 
+#define AS_FEATURE_BODY \
+	"<?xml version=\"1.0\" encoding=\"ISO-8859-1\"?>\
+<DoNotDisturbEvent xmlns=\"http://www.ecma-international.org/standards/ecma-323/csta/ed3\">\
```

Comment:
This URL returns a 404, seems that they changed their page structure

Reply:
I've used the same link as in presence_dfks module. I'm not sure what link should be used

Reply:
Ok, if its the same link - then lets keep it in sync.